### PR TITLE
Converted CHANGELOG from markdown to asciidoc

### PR DESCRIPTION
**Pull request by thatdocslady**
Fix for issue #407

Related to issue 557 in main ADB repository: 
https://github.com/projectatomic/adb-atomic-developer-bundle/issues/557

This issue is a part of the overall asciidoc migration and aims to unify the format of all doc files to asciidoc.

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -0,0 +1,275 @@
+= Revision History
+:toc:
+
+[[v1.3.3-sep-1-2016]]
+== 1.3.3 September 1, 2016
```
Comment (brgnepal) [inline CHANGELOG.adoc:5]: Why full month? we were using 3 character month previously

Comment (LalatenduMohanty) [reply CHANGELOG.adoc:5]: @budhrg I do not think this matter much. As long as we are consistent and can communicate  the time of release, we should be good.

**Comment by LalatenduMohanty** [comment]
LGTM

File: CHANGELOG.adoc (hunk 1 above)
Comment (thatdocslady) [reply CHANGELOG.adoc:5]: @budhrg we also did this in the main ADB repo, it's best practice to spell out months in docs and it's a part of the evolution of the doc set.

**Comment by hferentschik** [comment]
please test

**Comment by thatdocslady** [comment]
@hferentschik please test what?

**Comment by hferentschik** [comment]
@thatdocslady "please test" is a command to the CI instance. It is a response to the fact that you are not on the whitelist of people for whom the pull request are automatically validated.

**Comment by hferentschik** [comment]
add to whitelist

**Comment by hferentschik** [comment]
>  I do not think this matter much. As long as we are consistent and can communicate the time of release, we should be good.

+1 I don't think the markup language matters much here. What does matter though is to find a better format than just listing git commits. The changelog should be based on actual issues fixed and backed by milestones. Last but not least, we should generate it automatically. That's all on the roadmap thought ... ;-)

**Comment by hferentschik** [comment]
@thatdocslady that said, each pull request should have an underlying issue which allows us to track in in milestones and also generate a proper changelog further down the track. The commit message should then reference this issue. 

I'll take care of this, this time around :-)

**Comment by hferentschik** [comment]
Created issue #407, adjusted commit message, rebased and merged :-)

**Pull request closed without merging**